getMultiStarts.m

```diff
@@ -295,7 +318,7 @@
                 % parameters.constraints.A  ,parameters.constraints.b  ,... % linear inequality constraints
                 % parameters.constraints.Aeq,parameters.constraints.beq,... % linear equality constraints
                 
-                parameters.MS.J(1, i) = -J_0;
+                parameters.MS.J(1, i) = nan;
```

Comment:
then the naming is quite misleading, should'nt that be `parameters.MS.logPost0`?